src/gettext.cpp

```diff
@@ -216,30 +249,12 @@ void init_gettext(const char *path, const std::string &configured_language,
 #endif
 #endif
 
-	std::string name = lowercase(PROJECT_NAME);
-	infostream << "Gettext: domainname=\"" << name
-		<< "\" path=\"" << path << "\"" << std::endl;
-
-	bindtextdomain(name.c_str(), path);
-	textdomain(name.c_str());
+	// Notify gettext that the language has changed
+	// Source: https://www.gnu.org/software/gettext/manual/html_node/gettext-grok.html
+	{
+		extern int  _nl_msg_cat_cntr;
+		++_nl_msg_cat_cntr;
```

Comment:
I wonder if this won't mess up on systems where they are using some sort of alternative gettext implementation

Reply:
Whilst the documentation says to do this, I found that it wasn't actually needed.  So if it's likely to cause issues, could maybe be removed

Reply:
Related: https://bugs.kde.org/show_bug.cgi?id=365917 (fixed by https://invent.kde.org/frameworks/ki18n/-/commit/543b8ae67c57500bf3cabde75bf1599904d6e482) and also the https://github.com/minetest/minetest/pull/14379#issuecomment-1975327822 comment below.

Maybe only enable language switching when it can be determined that it works? (e.g. by checking whether translating LANG_CODE works after switching languages)